scripts/get-sbt

```diff
@@ -1,2 +1,2 @@
-curl -s https://raw.githubusercontent.com/paulp/sbt-extras/master/sbt > ./scripts/sbt \
```

Comment:
Any particular reason to change this?  I prefer to not modify the TPS scripts unless it is necessary.